### PR TITLE
Failing Test: Add Patient Appointment Test

### PR DESCRIPTION
**Pull request by insookwa**
Fixed the failing firefox tests 
kindly review PR for https://issues.openmrs.org/browse/RATEST-90
@kdaud   @sherrif10

### CODE REVIEW

**Comment by sherrif10** [comment]
Always use git pull upstream master to update your branch.

File: ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java [hunk 1]
```diff
@@ -57,7 +58,7 @@ public void addPatientAppointmentTest() throws Exception {
         findPatientPage = manageAppointmentsPage.saveAppointment();
         findPatientPage.enterPatient(patient.getName());
         manageAppointmentsPage = findPatientPage.clickOnFirstPatientAppointment();
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java:60]: Thanks @insookwa , Seems @kdaud  finished this same work and it was merged here https://github.com/openmrs/openmrs-distro-referenceapplication/commit/336813b1c8a9bd44239a0ba4f37a2702834f7aa1  . Probably this is a duplicate. Is there any difference.

**Review by sherrif10 — APPROVED** [review]
LGTM cc @dkayiwa  ,Thanks @insookwa . Well done man

File: ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java (hunk 1 above)
Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java:60]: AddPatientAppointment shares some logical steps with BookRequestAppointment test  though they both achive a different goal

**Review by kdaud — APPROVED** [review]
Great effort @insookwa !
I love the way your progressing! Keep fire burning !

cc: @dkayiwa

**Comment by kdaud** [comment]
@insookwa would be a good idea now to focus on taking on tasks concerning E2E Test to get you advance your skill set. How far have you gone with [RATEST-69 ](https://issues.openmrs.org/browse/RATEST-69) ? BTW did you have a chance to take a look on how to develop a new E2E Test in this [video](https://www.youtube.com/watch?v=jhJJAEBS9sQ&t=4s) ?

**Pull request merged**